File: store/src/test/java/com/dropbox/android/external/store4/Clearable.java
package com.dropbox.android.external.store4;


import javax.annotation.Nonnull;

/**
 * Persisters should implement Clearable if they want store.clear(key) to also clear the persister
 *
 * @param <T> Type of key/request param in store
 */
@Deprecated//used in tests
public interface Clearable<T> {
    void clear(@Nonnull T key);
}

File: store/src/test/java/com/dropbox/android/external/store4/Clearable.kt
package com.dropbox.android.external.store4

/**
 * Persisters should implement Clearable if they want store.clear(key) to also clear the persister
 *
 * @param <T> Type of key/request param in store
</T> */
@Deprecated("") //used in tests
interface Clearable<T> {
    fun clear(key: T)
}
